app/config/sources.inc.php

```diff
@@ -579,462 +49,1041 @@
 ];
 
 /*
-    Array structure for single website:
-    [
-       0 name,
-       1 path to local repo,
-       2 folder containing locale files,
-       3 array of supported locale,
-       4 array of supported file names,
-       5 reference locale,
-       6 url to public repo,
-       7 array of flags,
-       8 type of files (lang, raw)
-    ]
+    For each file it's possibile to specify the following fields:
+    * deadline: if the deadline is the same for all locales, assign the deadline
+      as a string in ISO format (dd-mm-yyyy):
+
+      'deadline' => '2016-04-29',
+
+      If deadline needs to change based on the locale, you can use a
+      multidimensional array, where each deadline is associated to a list of
+      locales. 'all' can be used to indicate that deadline applies to all
+      supported locales. Example, deadline only for two locales:
+
+      'deadline' => [
+          '2016-04-25' => ['de', 'fr'],
+      ],
+
+      Different deadlines:
+
+      'deadline' => [
+          '2016-04-25' => ['de', 'fr'],
+          '2016-01-05' => ['all'],
+      ],
+
+    * priority: priorities are numeric values assigned to files. 1 is the
+      highest priority, 3 is the lowest.
```

Comment:
We don't, it's just that I find Pontoon's classification extremely confusing. IMO if it's priority, 1 is more important than 5.

Since tiers use a similar system (1 -> 3), Pontoon will have to support mapping priority to stars anyway.

Alternatives are:
* Keep using this sytem
* Use priorities from 1 to 5, knowing that they're backwards compared to Pontoon
* Use the same system as Pontoon (5 -> 1)